packages/search/src/content/helpers/convert-request-to-portal-params.ts

```diff
@@ -181,6 +181,9 @@ function stringifyFilterValue(
 function decodePage(page: string): IPagingParams {
   try {
     const decodedPage: any = decode(page);
+    if (decodedPage === null) {
+      throw new Error();
```

Comment:
Is this for parity w/ `base-64`'s behavior? Should we have a message here. I'm fine to add a message later.

Reply:
Yes, it is. I'm just throwing to get down into the catch block where the message actually gets logged

Reply:
alternative would be
```js
if (!decodedPage) {
    return undefined;
}
```